djautotask/api.py

```diff
@@ -299,8 +299,10 @@ def __init__(
             rest_api_version = settings.AUTOTASK_CREDENTIALS[
                 'rest_api_version']
         if not server_url:
-            server_url = get_api_connection_url()
-
+            try:
+                server_url = get_api_connection_url()
+            except AutotaskAPIError as e:
+                raise AutotaskAPIError('{}'.format(e))
```

Comment:
Why is this catching and reraising the same exception?

Reply:
fixed